test/cpp/jit/test_gpu.cpp

```diff
@@ -1335,7 +1335,7 @@ TEST_F(NVFuserTest, FusionForLoop_CUDA) {
   BinaryOp* op = static_cast<BinaryOp*>(TV2->definition();
   fusion.addOutput(TV2);
 
-  auto fl = new kir::ForLoop(new kir::Int(c10::nullopt), ID0, {op});
+  auto fl = new kir::ForLoop(new Int(c10::nullopt), ID0, {op});
```

Comment:
I was puzzled why `kir::ForLoop` could be constructed this way without using the passkey, and just realized this is commented out.  Do we want to keep this test? Or is it considered stale?

Reply:
Test doesn't make much sense to me, removing.